### PR TITLE
Add domain-based routing to SIP Connect page

### PR DESCRIPTION
**Pull request by giacomovaccavonage**
## Description

SIP Connect has a new feature: domain-based routing.

## Deploy Notes

Nothing.

### CODE REVIEW

**Comment by mheap** [comment]
@tbedford Any thoughts on this?

File: _documentation/voice/sip/concepts/sip-connect.md [hunk 1]
```diff
@@ -18,3 +18,24 @@ To test this functionality have your PBX forward calls to `sip.nexmo.com`. Here
 exten => 69100,1,SIPAddHeader(X-UserId:ABC123)
 exten => 69100,2,Dial(SIP/nexmo/14155550100)
 ```
+
+## Domain-based routing
+
+If you are using `SIP Connect` and require your call to be handled by a specific Nexmo region, then you can specify that by using R-URI with such information in the domain part.
+
+e.g.:
```
Comment (tbedford) [inline _documentation/voice/sip/concepts/sip-connect.md]: Change "e.g.:" for "The following code will indicate to Nexmo that you want this `SIP Connect` call to be handled in the EU:"

File: _documentation/voice/sip/concepts/sip-connect.md [hunk 2]
```diff
@@ -18,3 +18,24 @@ To test this functionality have your PBX forward calls to `sip.nexmo.com`. Here
...
+```
+sip:anLVN@sip-eu1.nexmo.com
+```
+will instruct us that you want this `SIP Connect` call to be handled in EU.
```
Comment (tbedford) [inline _documentation/voice/sip/concepts/sip-connect.md]: Remove this line.

**Comment by tbedford** [comment]
Spell checker is also generating errors for:

```
using R-URI with
- sip-eu1.nexmo.com: EU
- sip-eu2.nexmo.com: EU
- sip-ap1.nexmo.com: AP
- sip-ap2.nexmo.com: AP
```

@fabianrbz is there a tweak we can do so this format can be handled, or do we need to add to `.spelling`.

**Comment by giacomovaccavonage** [comment]
> Spell checker is also generating errors for:
> 
> ```
> using R-URI with
> - sip-eu1.nexmo.com: EU
[...]

As far as I'm concerned, feel free to suggest a different format. It's just a domain name to be associated to a region.

**Comment by tbedford** [comment]
@mheap, @fabianrbz : This is good to merge now once the build issues (our side) are sorted:

```
error An unexpected error occurred: "https://registry.yarnpkg.com/@xtuc/long/-/long-4.2.2.tgz: Request failed \"504 Gateway Timeout\"".
```

**Pull request merged**